src/main/kotlin/com/londogard/summarize/embeddings/WordEmbeddings.kt

```diff
@@ -170,26 +166,22 @@ class WordEmbeddings(
             .fold(listOf<Array<Float>>() as List<Array<Float>>?) { agg, itr ->
                 agg?.let { lst ->
                     vector(itr)?.let { v ->
-                        listOf(
-                            v
-                        ) + lst
+                        listOf(v) + lst
                     }
                 }
             }?.reversed()
     }
```

Comment:
Kanske bättre namn på agg och itr?

Reply:
Makes sense, but agg = aggregator and is good to keep imo.
Itr aka iterator could be renamed into what it is (vector) or such 👍